### PR TITLE
change code blocks and info on bugs in functions

### PR DESCRIPTION
**Pull request by AlexandraKapp**
- Fixed: Code blocks were not rendered in the GitHub readme with :linenos: directive. 
- Import info in QuickStart not correct for current Version
- Info on Bug in allRegions() Query -> to be fixed in next Version

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -64,14 +62,13 @@ Within your python file or notebook:
 - either for single regions
 
 .. code-block:: python
-   :linenos:
 
     query = Query.region('01')
 
 - or for all subregions a region (e.g. all Kommunen in a Bundesland)
+(to be implemented in Version 0.1.2)
```
Comment (KonradUdoHannes) [inline README.rst]: Not sure whether this is a bug or a feature. (Our first bug feature discussion XD)
Feature would mean its added in 0.2.0

File: README.rst [hunk 2]
```diff
@@ -121,17 +114,18 @@ Get information on fields and meta data
 **Get information on region ids**
 
 .. code-block:: python
-   :linenos:
 
+    # Version >= 0.1.2
     from datenguidepy import get_all_regions
+    # Version < 0.1.2
+    from datenguidepy.query_helper import get_all_regions
```
Comment (KonradUdoHannes) [inline README.rst]: Does also not appear to be a bug to me, with respect to the versioning. Still true if our next release is simply 0.2.0 though.

File: README.rst [hunk 3]
```diff
@@ -140,9 +134,11 @@ For more information on "nuts" levels see Wikipedia_.
 **Get information on statistic shortnames**
 
 .. code-block:: python
-   :linenos:
 
+    # Version >= 0.1.2
     from datenguidepy import get_statistics
+    # Version < 0.1.2
+    from datenguidepy.query_helper import get_statistics
```
Comment (KonradUdoHannes) [inline README.rst]: Same as above. Not a bug in my opinion.

File: README.rst (hunk 1 above)
Comment (AlexandraKapp) [reply README.rst]: i would say bug, because its already in the docu of the Query

**Pull request merged**